services/distribution/api_v1.json

```diff
@@ -1,458 +1,457 @@
+                "tags": [
+                    "Diagnosis keys"
+                ],
+                "parameters": [
+                    {
+                        "name": "version",
+                        "in": "path",
+                        "description": "API version",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/components/schemas/Version"
+                        }
+                    },
+                    {
+                        "name": "country",
+                        "in": "path",
+                        "description": "Country",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/components/schemas/Country"
+                        }
+                    },
+                    {
+                        "name": "date",
+                        "in": "path",
+                        "description": "Date",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/components/schemas/Date"
+                        }
+                    },
+                    {
+                        "name": "hour",
+                        "in": "path",
+                        "description": "Hour",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/components/schemas/Hour"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "All diagnosis keys for that hour.",
+                        "content": {
+                            "application/zip": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/app.coronawarn.server.common.protocols.external.TemporaryExposureKeyExport"
+                                },
+                                "example": "<BINARY DATA>"
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/version/{version}/configuration/country/{country}/app_config": {
+            "get": {
+                "summary": "Get application configuration for a country.",
+                "description": "Gives application configuration for a specific country, which includes:\n- Minimum risk score\n- Risk score classification\n- Risk score parameters\n- Attenuation duration\n- Application version configuration",
```

Comment:
We should probably explain this in more detail. E.g. that the app_config is generated for the origin country, and update the values in the file (supported countries).